src/tests/Country.test.js

```diff
@@ -0,0 +1,18 @@
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/store';
+import Country from '../components/Country';
+
+describe('Country', () => {
+  it('should render the component', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Country name="Pakistan" flagEmoji="🇵🇰" continent="Asia" />
+        </BrowserRouter>
+      </Provider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Good job adding tests, however, please make sure to add tests that test the pure functions in your redux code as well as some integration tests for the components and not just snapshot tests, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.) requirement.